phpunit.xml.dist

```diff
@@ -1,20 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.0/phpunit.xsd"
```

Comment:
Why change this?

Reply:
I used the migration that is shipped with phpunit. If the config has incompatibilities between versions, you can adapt it with `vendor/bin/phpunit --migrate-configuration`

Reply:
Great, but I doubt we are going to remember to bump this number when upgrading to 10.1, so I'd revert.

Reply:
I don't know if phpunit would also invoke a warning in that case. I reverted it back.